examples-native/crna-kitchen-sink/.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["babel-preset-expo"],
+  "presets": ["react-native"],
```

Comment:
The only question I have what was changed here?

Reply:
`npm run server` is broken right now because of this config. I just removed it from here. I'm to open a PR with it.

Reply:
@elucaswork  Do you know why is it broken?

Reply:
It was an issue on `expo` `30.0.1`

It fails with `npm run storybook`
<img width="1368" alt="screen shot 2018-11-07 at 8 40 55 am" src="https://user-images.githubusercontent.com/20709038/48134896-5b89c880-e269-11e8-9a66-bfca9da43be8.png">

But I just checked out that it's fixed on `31.0.4`. So #4728 will address it.